bin/ovpn_initpki

```diff
@@ -18,6 +18,9 @@ nopass=$1
 # Provides a sufficient warning before erasing pre-existing files
 easyrsa init-pki
 
+# Create a random file for easyrsa rng
+dd if=/dev/urandom of=$EASYRSA_PKI/.rnd bs=256 count=1
```

Comment:
What's the intended lifetime of this file?  Is there no way for EasyRSA to use the pseudo-random number generator directly instead of persisting randomness?